File: 프로그래머스/1/147355. 크기가 작은 부분문자열/크기가 작은 부분문자열.py
def solution(t, p):
    answer = 0
    l = len(p)
    
    for i in range(0,len(t)):
        s = t[i:i+l]
        
        if len(s) < l:
            break;
        
        if s <= p:
            answer += 1
    
    return answer

File: 프로그래머스/1/147355. 크기가 작은 부분문자열/크기가 작은 부분문자열.js
function solution(t, p) {
    var answer = 0;
    let len = p.length;
    
    for(var i=0; i<t.length; i++){
        let s = t.slice(i,i+len)
        
        if (s.length < len){
            break;
        }
        if(Number(s)<=Number(p)){
            answer += 1
        }
    }
    
    return answer;
}